### ISSUE TITLE
LO-WSRPN

### ISSUE THREAD

**Issue by cpy0029** [HunterJ-Lin/WSOVOD#5]
I have output the model structure, but I didn't see the LO-WSRPN mentioned in the article. Instead, there is one 3*3 convolution and two 1*1 convolutions.

(proposal_generator): WSOVODRPN_V2(
    (rpn_head): StandardRPNHead(
      (conv): Conv2d(
        512, 512, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1)
        (activation): ReLU()
      )
      (objectness_logits): Conv2d(512, 18, kernel_size=(1, 1), stride=(1, 1))
      (anchor_deltas): Conv2d(512, 72, kernel_size=(1, 1), stride=(1, 1))
    )
    (anchor_generator): DefaultAnchorGenerator(
      (cell_anchors): BufferList()
    )
  )

Where is the code for LO-WSRPN located? Could it be that I'm looking in the wrong place?
Thanks！

**Comment by HunterJ-Lin** [comment]
We have found the new LO-WSRPN, i.e. WSOVODRPN_V2 get higher mAP in close-set weakly supervised object detection, and comparable AP noval in open-vocabulary. We have released the two version in the code, and we will renew the LO-WSRPN after mix-dataset training code released.

**Comment by cpy0029** [comment]
In the paper, it mentions one 3*3 convolution and three 1*1 convolutions. In the code, where are these located?
